### PR TITLE
[5.1] Let vendor:publish handles an array of tags

### PR DESCRIPTION
**Pull request by lucasmichot**
Replace #9362

### CODE REVIEW

File: src/Illuminate/Foundation/Console/VendorPublishCommand.php [hunk 1]
```diff
@@ -53,12 +53,22 @@ public function __construct(Filesystem $files)
      */
     public function fire()
     {
+        foreach ($this->option('tag') as $tag) {
+            return $this->publishTag($tag);
+        }
+    }
+
+    /**
+     * @param $tag
```
Comment (taylorotwell) [inline src/Illuminate/Foundation/Console/VendorPublishCommand.php]: Can you put a full doc block on this.

Comment (lucasmichot) [inline src/Illuminate/Foundation/Console/VendorPublishCommand.php]: Done with https://github.com/lucasmichot/framework/commit/dcbc43a0c7c7ed32159b1ac5d3fc48e2ffad03d4

File: src/Illuminate/Foundation/Console/VendorPublishCommand.php [hunk 2]
```diff
@@ -53,12 +53,25 @@ public function __construct(Filesystem $files)
      */
     public function fire()
     {
+        foreach ($this->option('tag') as $tag) {
+            return $this->publishTag($tag);
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/VendorPublishCommand.php]: really return?

Comment (lucasmichot) [inline src/Illuminate/Foundation/Console/VendorPublishCommand.php]: Course no

**Pull request merged**